### PR TITLE
configured symfony app with docker

### PR DESCRIPTION
**Pull request by arslantoor**

### CODE REVIEW

File: .dockerignore [hunk 1]
```diff
@@ -0,0 +1,12 @@
+./app/var
```
Comment (adeelnawaz) [inline .dockerignore]: Why ./app?

File: .dockerignore [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+/var/
+/vendor/
+
+###< symfony/framework-bundle ###
+###> symfony/phpunit-bridge ###
+/phpunit.xml.dist
+/phpunit.xml
+phpunit.xml.dist
+docker-compose.yml.local
```
Comment (adeelnawaz) [inline .dockerignore]: Change this to docker-compose.yml

File: .dockerignore [hunk 3]
```diff
@@ -0,0 +1,15 @@
+.git
+*.md
+
+.idea
+.idea/*
```
Comment (adeelnawaz) [inline .dockerignore]: This is redundant

File: .dockerignore [hunk 4]
```diff
@@ -0,0 +1,15 @@
+.git
```
Comment (adeelnawaz) [inline .dockerignore:2]: Following files are missing:
- doceker-compose.yml.dist
- doceker-compose.yml
- Dockerfile

File: .dockerignore [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+.idea/*
+
+/var/
+/vendor/
+
+###< symfony/framework-bundle ###
+###> symfony/phpunit-bridge ###
+/phpunit.xml.dist
+/phpunit.xml
+phpunit.xml.dist
```
Comment (adeelnawaz) [inline .dockerignore:10]: Remove duplications

File: .gitignore [hunk 6]
```diff
@@ -0,0 +1,18 @@
+### symfony/framework ###
```
Comment (adeelnawaz) [inline .gitignore:1]: Remove all duplications

File: Dockerfile [hunk 7]
```diff
@@ -0,0 +1,7 @@
+FROM richarvey/nginx-php-fpm:1.10.3
+
+ENV WEBROOT="/var/www/html/public/"
+
+COPY . /var/www/html/
```
Comment (adeelnawaz) [inline Dockerfile]: Set proper permissions here
```
COPY --chown nginx:nginx . /var/www/html/
```

File: README.md [hunk 8]
```diff
@@ -1,2 +1 @@
-# php-symfony-task
-# php-symfony
+# php-symfony with docker
```
Comment (adeelnawaz) [inline README.md]: Add the following
- Description
- Prerequisites
- Setup steps

**Review by adeelnawaz — CHANGES_REQUESTED** [review]

File: .dockerignore (hunk 1 above)
Comment (arslantoor) [reply .dockerignore]: this file is update, that issue was when we have project in app dir and Dockerfile in nginx dir so this issue is fixed , but its  still he until i have merged base branch into it?

File: .dockerignore (hunk 5 above)
Comment (arslantoor) [reply .dockerignore:8]: this also updated and merged with its base branch, but its still here?

**Review by adeelnawaz — APPROVED** [review]

**Pull request merged**